siliconcompiler/remote/server.py

```diff
@@ -22,33 +25,34 @@ class Server:
     asynchronous siliconcompiler jobs, by using the slurm HPC daemon to
     schedule work on available compute nodes. It can also be configured to
     launch new compute nodes in the cloud, for on-demand jobs.
-
     """
 
+    __version__ = '0.0.1'
+
     ####################
-    def __init__(self, cmdlinecfg, loglevel="DEBUG"):
+    def __init__(self, loglevel="INFO"):
         '''
         Init method for Server object
-
         '''
 
         # Initialize logger
-        self.logger = log.getLogger(uuid.uuid4().hex)
-        self.handler = log.StreamHandler()
-        self.formatter = log.Formatter('%(asctime)s %(levelname)-8s %(message)s')
-        self.handler.setFormatter(self.formatter)
-        self.logger.addHandler(self.handler)
+        self.logger = log.getLogger(f'sc_server_{id(self)}')
+        handler = log.StreamHandler(stream=sys.stdout)
+        formatter = log.Formatter('%(asctime)s | %(levelname)-8s | %(message)s')
+        handler.setFormatter(formatter)
+        self.logger.addHandler(handler)
         self.logger.setLevel(loglevel)
 
-        # Use the config that was passed in.
-        self.cfg = cmdlinecfg
-        # Ensure that NFS mounting path is absolute.
-        nfsmount_abspath = [os.path.abspath(nfs_path) for nfs_path in self.cfg['nfsmount']['value']]
-        self.cfg['nfsmount']['value'] = nfsmount_abspath
+        self.schema = ServerSchema(logger=self.logger)
 
         # Set up a dictionary to track running jobs.
         self.sc_jobs = {}
 
+    def run(self):
+        if not os.path.exists(self.nfs_mount):
+            raise FileNotFoundError(f'{self.nfs_mount} could not be found.')
```

Comment:
Could this be a warning, with the directory being created if it doesn't exist?

`nfs_mount` was probably not a great choice for the parameter name; local directories also work when the server is executing jobs locally, as in our CI tests. It might make sense to change the parameter name to something like `jobs_dir` or `job_storage`.

Reply:
ohh, nevermind, I see what you mean, I think as part of reworking the server we can modify this behavior.